setup.py

```diff
@@ -144,7 +144,7 @@
     "pytest",
     "pytest-timeout",
     "pytest-xdist",
-    "python>=3.7.0",
+    "python>=3.7.1",
```

Comment:
Don't review this file. It's just for running (only) pipeline tests - I will remove the change before merge.